ci/prepare.sh

```diff
@@ -65,14 +65,6 @@ sudo apt-get remove -y ansible
 sudo pip install ansible==2.0.2.0
 sudo pip install --upgrade setuptools
 
-# Prepate the docker-ovn-hosts file
-LOCALIP=$(ip addr show dev eth0 | grep 'inet ' | cut -d " " -f 6 | cut -d "/" -f 1)
-if [ "$LOCALIP" == "" ] ; then
-    # Try bond0
-    LOCALIP=$(ip addr show dev bond0 | grep 'inet ' | cut -d " " -f 6 | cut -d "/" -f 1)
-fi
-cat ansible/docker-ovn-hosts-example | sed -e "s/REPLACE_IP/$LOCALIP/g" > ansible/docker-ovn-hosts
```

Comment:
With these line removed, how can the task discover the IP of the host in the ci job? Thanks.